sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/source/CosmosChangeFeedStartFromModes.java

```diff
@@ -19,7 +19,7 @@ public String getName() {
 
     public static CosmosChangeFeedStartFromModes fromName(String name) {
         for (CosmosChangeFeedStartFromModes startFromModes : CosmosChangeFeedStartFromModes.values()) {
-            if (startFromModes.getName().equals(name)) {
+            if (startFromModes.getName().equalsIgnoreCase(name)) {
```

Comment:
Since `ChangeFeedMode` and `ChangeFeedStartFromModes` are now used across spark and kafka, may be we should have these in cosmos java SDK, so we can re-use them. 
For future, we might integrate them with spring data also. 
nothing blocking, but just brainstorming.

Reply:
I kind think it is better to decouple here -
   this is contract between kafka connector -> azure cosmos

for example -> when we added a new support in azure-cosmos, it does not necessarily mean it will be automatically supported in kafka. Separating feels more clear. But open for it

Reply:
Totally makes sense. Thanks @xinlian12